Source/Scene/Model.js

```diff
@@ -1440,6 +1454,14 @@ define([
         parseNodes(model);
     }
 
+    function usesExtension(model, extension) {
```

Comment:
Instead of doing this linear search (`indexOf`) over and over again when we call this function, we could instead create an `_extensionsUsed` property that walks through the array and creates a hashmap (just make an empty object to start, `{}`, and add properties to it).